src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -117,4 +117,7 @@ def generate_strategy_pool():
     if strategy_tuple.name in chosen_strategies:
       pool.add_strategy(strategy_tuple)
 
+  if do_strategy(strategy.MUTATOR_PLUGIN_STRATEGY):
```

Comment:
nit: a comment here would be helpful, like "Always include mutator plugin strategy because ...".